src/generic_ui/polymer/cloud-install.html

```diff
@@ -90,44 +87,20 @@
 
       <div class='content'>
         <div class='imgWrapper'>
-          <img src='../../icons/cloud/create-circle-ic.svg'>
+          <img src='../../icons/cloud/uproxy+digitalocean.svg'>
         </div>
         <h1>{{ 'CLOUD_INSTALL_GET_STARTED_TITLE' | $$ }}</h1>
         <p id='getStartedMessage'></p>
-        <uproxy-button raised on-tap='{{ showDigitalOceanAccountHelpOverlay }}'
-          id="newAccount">
-          {{ 'CREATE_A_NEW_ACCOUNT' | $$ }}
-        </uproxy-button>
-        <uproxy-button raised on-tap='{{ showLoginOverlay }}' id="haveAccount">
-          {{ 'I_HAVE_AN_ACCOUNT' | $$ }}
-        </uproxy-button>
-        <p id='existingServerMessage'></p>
-      </div>
-    </core-overlay>
-
-    <core-overlay id='digitalOceanAccountHelpOverlay'>
-      <uproxy-app-bar on-go-back='{{ back }}' color='#34AFCE'>
-        {{ 'CREATE_A_CLOUD_SERVER' | $$ }}
-      </uproxy-app-bar>
-
-      <div class='content'>
-        <div class='imgWrapper'>
-          <img src='../../icons/cloud/getstarted-step1.svg'>
-          <img src='../../icons/cloud/getstarted-step2.svg'>
-          <img src='../../icons/cloud/getstarted-step3.svg'>
-        </div>
-        <h1>{{ 'CLOUD_INSTALL_CREATE_ACCOUNT_TITLE' | $$ }}</h1>
-        <!-- TODO: some of this text needs to be bold - need to set the HTML element rather than use $$ to translate -->
-        <p id='createAccountMessage'></p>
-
         <paper-checkbox id='havePromoCode' label="{{ 'HAVE_PROMO_CODE' | $$ }}" on-change="{{havePromoChanged}}"></paper-checkbox>
-
         <uproxy-button raised on-tap='{{ launchDigitalOceanSignup }}'>
           {{ 'CREATE_A_NEW_ACCOUNT' | $$ }}
         </uproxy-button>
         <uproxy-button raised on-tap='{{ showLoginOverlay }}'>
-          {{ 'SIGN_ME_IN' | $$}}
+          {{ 'I_HAVE_AN_ACCOUNT' | $$ }}
         </uproxy-button>
+        <hr>
+        <h1>{{ 'CLOUD_INSTALL_EXISTING_SERVER_TITLE' | $$ }}</h1>
```

Comment:
Can we make the `Learn how` work like `Learn more.`, above? I know it's polish but it should be easy to get right: as-is, it's too easy to get bad wrapping, e.g.:

![image](https://cloud.githubusercontent.com/assets/5142116/16316578/39f66918-3955-11e6-9546-2b88236c2885.png)